downloadDirectory/src/test/java/org/reactome/release/downloadDirectory/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package org.reactome.release.downloadDirectory;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
```

Comment:
This file can either be deleted, or filled with real unit tests (real unit tests would probably be a good idea).